functions/daemon.go

```diff
@@ -188,6 +210,15 @@ func setSubscriptions(client mqtt.Client, node *config.Node) {
 		logger.Log(0, "network", node.Network, token.Error().Error())
 		return
 	}
+	logger.Log(3, fmt.Sprintf("subscribed to proxy updates  /%s/%s", node.Network, node.ID))
+	if token := client.Subscribe(fmt.Sprintf("proxy/%s/%s", node.Network, node.ID), 0, mqtt.MessageHandler(ProxyUpdate)); token.WaitTimeout(mq.MQ_TIMEOUT*time.Second) && token.Error() != nil {
```

Comment:
if proxy settings are moved to the host, this topic could be updated to the host ID